outlines/models/vllm.py

```diff
@@ -185,3 +186,41 @@ def vllm(model_name: str, **vllm_model_params):
     model = LLM(model_name, **vllm_model_params)
 
     return VLLM(model)
+
+
+def adapt_tokenizer(tokenizer: PreTrainedTokenizerBase) -> PreTrainedTokenizerBase:
```

Comment:
Since vLLM uses `transformers` tokenizer this logic is repeated over from `transformers.py`. We need to re-use what is implemented in the latter.